index.d.ts

```diff
@@ -56,3 +56,10 @@ export function createCustomEqual<Meta = undefined>(
 export function createCustomEqual<Meta = WeakMap<any, any>>(
   getComparatorOptions: GetComparatorOptions<Meta>,
 ): EqualityComparator<Meta>;
+
+export function createCustomCircularEqual<Meta = undefined>(
```

Comment:
I believe this is actually an issue with copy pasta that has been propagated with more copy pasta! Above I have two implementations of `createCustomEqual`, however the second should be `createCustomCircularEqual`. If you simply fix the above to be this, it should suffice without adding more entries:

```ts
export function createCustomEqual<Meta = undefined>(
  getComparatorOptions: GetComparatorOptions<Meta>,
): EqualityComparator<Meta>;
export function createCustomCircularEqual<Meta = WeakMap<any, any>>(
  getComparatorOptions: GetComparatorOptions<Meta>,
): EqualityComparator<Meta>;
```